.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor
 composer.lock
 reports
+/.idea
```

Comment:
Can we remove this line? While there's competing arguments, I elect to having IDE specific files reside in the global gitignore.